### PR TITLE
Dev 12.0 afc 1598 mer översättningar

### PR DESCRIPTION
**Pull request by timforsstromAF**

### CODE REVIEW

File: outplacement_partner_education/i18n/sv_SE.po [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+#. module: outplacement_partner_education
+#: model:ir.model.fields,field_description:outplacement_partner_education.field_outplacement__cv
+msgid "CV"
+msgstr ""
+
+#. module: outplacement_partner_education
+#: model:ir.model.fields,field_description:outplacement_partner_education.field_outplacement__cv_file_name
+msgid "Cv File Name"
+msgstr ""
```
Comment (arturchilangwa-vertel) [inline outplacement_partner_education/i18n/sv_SE.po:26]: CV filnamn

File: outplacement/i18n/sv_SE.po [hunk 2]
```diff
@@ -0,0 +1,539 @@
...
+
+#. module: outplacement
+#: model:ir.model.fields,field_description:outplacement.field_outplacement__activity_ids
+msgid "Activities"
+msgstr "Aktiviteter"
+
+#. module: outplacement
+#: model:ir.model.fields,field_description:outplacement.field_outplacement__activity_state
+msgid "Activity State"
+msgstr ""
```
Comment (arturchilangwa-vertel) [inline outplacement/i18n/sv_SE.po:31]: Should this be empty?

File: outplacement_partner_education/i18n/sv_SE.po (hunk 1 above)
Comment (arturchilangwa-vertel) [inline outplacement_partner_education/i18n/sv_SE.po:26]: Maybe it should be called: CV filnamn?

File: outplacement_partner_education/i18n/sv_SE.po [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+#. module: outplacement_partner_education
+#: model:ir.model.fields,field_description:outplacement_partner_education.field_outplacement__references
+msgid "References"
+msgstr "Referenser"
+
+#. module: outplacement_partner_education
+#: model:ir.model.fields,field_description:outplacement_partner_education.field_outplacement__references_file_name
+msgid "References File Name"
+msgstr ""
```
Comment (arturchilangwa-vertel) [inline outplacement_partner_education/i18n/sv_SE.po:76]: Filnamn för referenser? Maybe, I am unsure.

**Review by arturchilangwa-vertel — CHANGES_REQUESTED** [review]
Some of the fields are empty and I dont know if they be that. 
I have successfully compiled these files with `pocompile`, is there an other way to test the transaltions?

**Comment by MickeHolm** [comment]
Included in https://github.com/vertelab/odoo-outplacement/pull/36
So closing tihs

**Pull request closed without merging**